### DISCUSSION TITLE
How do I get the percentage of row to sum of column?

### DISCUSSION THREAD

**Discussion by UrDub** [saulpw/visidata#912] [Q&A]
I would like to create a new column that will show the percentage of each row of a column with the sum of that column. If the column name is "february", I would like to do the following:

`new column expr=february/<sum-of-february>`

what should I place in `<sum-of-february>`?

**Comment by geekscrapy** [comment]
If I understand you correctly you want to add up all the float values in a column? If so, use `z+` and "sum" when on the column you want to add up. This wil show the sum of the column in the status bar. Hope that helps!

**Reply by UrDub** [reply]
that is right, the sum of the column. however, instead of doing `z+` and typing in the result in 
`new column expr=february/<columnsum>`, 
I would like to do it programatically, like 
`new column expr=february/february.sum.` 
Does visidata have a method or variable that corresponds to `february.sum`?

**Comment by gennaro-tedesco** [comment]
There is a somewhat similar solution [here](https://github.com/saulpw/visidata/discussions/2490#discussion-7019776), though I agree there ought to be a simple `column.sum` method to define new columns with aggregated values on the fly (or perhaps there is, and we could not find it in the docs).
